controller/backup_controller.go

```diff
@@ -328,6 +328,12 @@ func (bc *BackupController) reconcile(backupName string) (err error) {
 			bc.syncBackupStatusWithSnapshotCreationTimeAndVolumeSize(volume, backup)
 		}
 
+		defer func() {
+			if backup.Status.State == longhorn.BackupStateError || backup.Status.State == longhorn.BackupStateUnknown {
+				backup.Status.LastSyncedAt = syncTime
+			}
+		}()
```

Comment:
This is seems like a change in functionality, you are now always resetting the syncTime, when the `checkMonitor/syncWithMonitor` throws an error.

Are there any repercussions for this?

Reply:
Revert it. We should set the `status.lastSyncedAt` once the backup monitor enablement failed.